Tests/NIOTransportServicesTests/NIOTSBootstrapTests.swift

```diff
@@ -336,6 +336,29 @@ final class NIOTSBootstrapTests: XCTestCase {
             try? connectionChannel?.close().wait()
         }
     }
+
+    func testBootstrapsMultipath() throws {
+        let group = NIOTSEventLoopGroup()
+        defer {
+            try! group.syncShutdownGracefully()
+        }
+
+        let listenerBootstrap = NIOTSListenerBootstrap(group: group).withMultipath(.handover)
+        let connectionBootstrap = NIOTSConnectionBootstrap(group: group).withMultipath(.handover)
+
+        do{
+            let listenerChannel: Channel = try listenerBootstrap.bind(host: "localhost", port: 0).wait()
+            let connectionChannel: Channel = try connectionBootstrap.connect(to: listenerChannel.localAddress!).wait()
+            defer{
+                try? listenerChannel.close().wait()
+                try? connectionChannel.close().wait()
+            }
+            XCTAssertEqual(try listenerChannel.getOption(NIOTSChannelOptions.multipathServiceType).wait(), .handover)
+            XCTAssertEqual(try connectionChannel.getOption(NIOTSChannelOptions.multipathServiceType).wait(), .handover)
+        }catch{
+            XCTFail()
+        }
```

Comment:
We don't need this do-catch, the test will fail if it throws an error.

Reply:
Indeed !